app/components/remove-log-popup.js

```diff
@@ -17,7 +17,7 @@ export default Ember.Component.extend({
 
       return job.removeLog().then(() => {
         this.get('flashes').success('Log has been successfully removed.');
-      }).fail((xhr) => {
+      }, (xhr) => {
```

Comment:
I just reverted to what it was before, though I think it might be clearer if we used `catch`/`error`/`fail` in the future instead of the implicit failure. We seem to use this pattern in multiple places atm.